changelogs/unreleased/6546-chaosbox-small.md

```diff
@@ -0,0 +1 @@
+Add "compression" flag to set/disable the compression type applied in the default HTTP filter chain.
```

Comment:
this might need update as well based on the new api

Reply:
included in [5dcdc33](https://github.com/projectcontour/contour/pull/6546/commits/5dcdc33aeb0d1b27c24527e34734857ea2583612)